R/class_clv_bgnbd_staticcov.R

```diff
@@ -0,0 +1,37 @@
+#' Result of fitting the BG/NBD model with static covariates
+#'
+#' @description
+#' Output from fitting the BG/NBD model on data with static covariates. It constitutes the
+#' estimation result and is returned to the user to use it as input to other methods such as
+#' to make predictions or plot the unconditional expectation.
+#'
+#' Inherits from \code{clv.fitted.static.cov} in order to execute all steps required for fitting a model
+#' with static covariates and it contains an instance of class \code{clv.model.bgnbd.static.cov} which
+#' provides the required BG/NBD (static covariates) specific functionalities.
+#'
+#' @template template_slot_bgnbdcbs
+#'
+#' @seealso \link{clv.fitted.static.cov-class}, \link{clv.model.bgnbd.static.cov-class}, \link{clv.bgnbd-class}
+#'
```

Comment:
make a template for all static cov result classes and use templateVar to adjust the model names (BGNBD vs PNBD). this is newly done for the nocov clv.pnbd, see there for an example.